src/components/inputs/localized-money-input/localized-money-input.spec.js

```diff
@@ -576,110 +576,54 @@ describe('LocalizedMoneyInput.parseMoneyValues', () => {
+          USD: '2.001',
+          EGP: '12.00',
+        })
+      ).toMatchObject(['USD']);
     });
   });
   describe('when called with an empty money value', () => {
     it('should throw', () => {
       expect(() =>
-        LocalizedMoneyInput.isHighPrecision([
+        LocalizedMoneyInput.getHighPrecisionCurrencies([
           { amount: '', currencyCode: 'EUR' },
         ])
       ).toThrow();
     });
   });
 });
 
-describe('LocalizedMoneyInput.isEmpty', () => {
-  describe('when value is filled out', () => {
-    it('should return false for empty currencies', () => {
-      expect(
-        LocalizedMoneyInput.isEmpty([{ amount: '5', currencyCode: 'EUR' }])
-      ).toMatchObject({ EUR: false });
-    });
-  });
-  describe('when value is empty', () => {
-    it('should return true', () => {
-      expect(
-        LocalizedMoneyInput.isEmpty([{ amount: '', currencyCode: 'EUR' }])
-      ).toMatchObject({ EUR: true });
-    });
-  });
-});
-
-describe('LocalizedMoneyInput.isAllEmpty', () => {
-  describe('when value is filled out', () => {
-    it('should return false if some have value', () => {
-      expect(
-        LocalizedMoneyInput.isAllEmpty([
-          { amount: '5', currencyCode: 'EUR' },
-          { amount: '', currencyCode: 'EUR' },
-        ])
-      ).toBe(false);
-    });
-  });
-  describe('when value is empty', () => {
-    it('should return true if all are empty', () => {
-      expect(
-        LocalizedMoneyInput.isAllEmpty([
-          { amount: '', currencyCode: 'EUR' },
-          { amount: '', currencyCode: 'EUR' },
-        ])
-      ).toBe(true);
-    });
-  });
-});
-
-describe('LocalizedMoneyInput.hasEmptyValue', () => {
+describe('LocalizedMoneyInput.getEmptyCurrencies', () => {
   describe('when value is filled out', () => {
-    it('should return false if some have value', () => {
+    it('should return empty array', () => {
       expect(
-        LocalizedMoneyInput.hasEmptyValue([
-          { amount: '5', currencyCode: 'EUR' },
-          { amount: '', currencyCode: 'EUR' },
-        ])
-      ).toBe(true);
+        LocalizedMoneyInput.getEmptyCurrencies({ EUR: '5.22', USD: '31.88' })
+      ).toMatchObject([]);
     });
   });
   describe('when value is empty', () => {
-    it('should return true if all are empty', () => {
+    it("should return the currencies that don't have value", () => {
       expect(
-        LocalizedMoneyInput.hasEmptyValue([
-          { amount: '', currencyCode: 'EUR' },
-          { amount: '', currencyCode: 'EUR' },
-        ])
-      ).toBe(true);
-    });
-  });
-  describe('when value is all', () => {
-    it('should return true if all are empty', () => {
-      expect(
-        LocalizedMoneyInput.hasEmptyValue([
-          { amount: '21.33', currencyCode: 'EUR' },
-          { amount: '51.99', currencyCode: 'EUR' },
-        ])
-      ).toBe(false);
+        LocalizedMoneyInput.getEmptyCurrencies({ EUR: '', USD: '31.88' })
+      ).toMatchObject(['EUR']);
```

Comment:
`toEqual` would work, right? or `toEqual(expect.arrayContaining([])` if you need it more flexible.